content/20.pharmaceuticals.md

```diff
@@ -44,6 +44,7 @@ SARS-CoV-1 was officially contained on July 5, 2003, within 9 months of its appe
 In contrast, SARS-CoV-2 remains a significant global threat nearly two years after its emergence.
 COVID-19 data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series; @doi:10/ggnsjk].
 SARS data are from the WHO [@url:https://www.who.int/csr/sars/country/en] and were obtained from a dataset on GitHub [@url:https://github.com/imdevskp/sars-2003-outbreak-data-webscraping-code].
+See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
```

Comment:
Very nice.  Can you add this to the clinical trials figure as well please?

Reply:
Done! Thanks for pointing that out!